components/authz-service/storage/v1/postgres/postgres.go

```diff
@@ -67,7 +67,7 @@ func (m *policyMap) Scan(src interface{}) error {
 
 // New instantiates the postgres IAM v1 storage backend.
 func New(ctx context.Context, l logger.Logger, migConf migration.Config, dataMigConf datamigration.Config) (storage.Storage, error) {
-	l.Infof("applying database migrations from %s", migConf.Path)
+	l.Infof("[v1] applying database migrations from %s", migConf.Path)
```

Comment:
Adding this vital distinction makes it obvious where the process starts. The v2 one does not have the "[v2]" tag so you can distinguish them.

Reply:
Do we need this change if v1 is going away? Or is it here for the interim to clarify things?

Reply:
The latter--that is what clued me in to the fact that the v1 server was running migrations.